src/logger/loggerTaskEx.c

```diff
@@ -243,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
Looked at it carefully and its part of the RC.  If it breaks we will know
it.  But could also be convinced to move it to 2.8.6.  Don't see much
difference IMHO.  I tested it locally and it worked fine.

On Thu, Sep 24, 2015 at 2:46 PM, Brent Picasso <notifications@github.com>
wrote:

> In src/logger/loggerTaskEx.c
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/89#discussion_r40363604>
> :
>
> > @@ -243,8 +247,7 @@ void loggerTaskEx(void *params)
> >                  }
> >
> >                  /* send the sample on to the telemetry task(s) */
> > -                if (sampledRate >= telemetrySampleRate ||
> > -                    currentTicks % telemetrySampleRate == 0)
> > +                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
>
> Changing this particular logic may cause a regression. Suggest moving to
> 2.8.6 where we can look at it more carefully.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/89/files#r40363604>
> .
>



--